tensorflow/lite/delegates/external/external_delegate.h

```diff
@@ -46,6 +46,11 @@ TfLiteDelegate* TfLiteExternalDelegateCreate(
 // Destroys a delegate created with `TfLiteExternalDelegateCreate` call.
 void TfLiteExternalDelegateDelete(TfLiteDelegate* delegate);
 
+// Insert key/value to the options.
+TfLiteStatus TfLiteExternalDelegateOptionsInsert(
```

Comment:
Move this declaration to line 37 (to match the order of the definitions in the .cc file)

Reply:
Fixed. Thank you.